playbook-website/app/javascript/site_styles/_opacity_guidelines.scss

```
@@ -0,0 +1,48 @@
+ul {
+   
+  &.opacity-play {
+  list-style: none;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: none;
+  background: none;
+  border-radius: none;
+  border: 0px;
+  padding: none;
+  flex-wrap: wrap;
+  }
+}
+
+.selector { 
+  .opacity_1{
+    opacity: $opacity_1;
+  }
+  .opacity_2{
```

```suggestion
  .opacity_2 {
```
You get the picture 😉